forwarder/forwarder.go

```diff
@@ -236,6 +239,11 @@ func (f *Forwarder) getClient(req *http.Request) (*http.Client, *url.URL) {
 		}, url
 	} else {
 		f.logger.Printf("Forwarding to %s", applicationURL.String())
-		return &http.Client{}, url
+		// allow for
+		// https://stackoverflow.com/questions/12122159/how-to-do-a-https-request-with-bad-certificate
+		tr := &http.Transport{
+			TLSClientConfig: &tls.Config{InsecureSkipVerify: f.sslSkipVerify},
```

Comment:
This looks OK as today we only have a single option. For the future, we’ll want to pass the tls config object in the forwarder struct directly, that’ll allow us to drive any tls config options from clammit configuration without adding more fields to the Forwarder struct.

Reply:
I agree - I was not happy also. But I did not wanted to do a full refactoring ....